examples/app.py

```diff
@@ -0,0 +1,147 @@
+# SPDX-License-Identifier: MIT
+# Copyright (C) 2024 Avnet
+# Authors: Nikola Markovic <nikola.markovic@avnet.com> et al.
+# This is a self-updating app with OTA support that can update itself with a new OTA package.
+
+import random
+import sys
+import time
+from dataclasses import dataclass
+import subprocess
+import os
+import urllib.request
+
+
+from avnet.iotconnect.sdk.lite import Client, DeviceConfig, C2dCommand, Callbacks, DeviceConfigError
+from avnet.iotconnect.sdk.lite import __version__ as SDK_VERSION
+from avnet.iotconnect.sdk.sdklib.mqtt import C2dAck, C2dOta
+
+"""
+See minimal.py example for a way to configure the device without the JSON file
+You can download the iotcDeviceConfig.json by clicking the cog icon in the upper right of your device's info panel
+NOTE: If you do not pass the server certificate, we will use the system's trusted certificate store, if available.
+For example, the trusted Root CA certificates from the in /etc/ssl/certs will be used on Linux.
+However, it is more secure to pass the actual server CA Root certificate in order to avoid potential MITM attacks.
+On Linux, you can use server_ca_cert_path="/etc/ssl/certs/DigiCert_Global_Root_CA.pem" for Azure
+or server_ca_cert_path="/etc/ssl/certs/Amazon_Root_CA_1.pem" for AWS
+"""
+
+@dataclass
+class ExampleAccelerometerData:
```

Comment:
Residue from a different file. We are just sending random number. If another example has this issue, fix it as well.